scripts/release/pr_description.md

```diff
@@ -0,0 +1,33 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- Pull request titles must use the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) format -->
+
+## Issue being fixed or feature implemented
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+Release new Dash Platform version
+
+## What was done?
+<!--- Describe your changes in detail -->
+- Updated changelog
+- Bumped packages version
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+None
+
+## Breaking Changes
+<!--- Please describe any breaking changes your code introduces and verify that -->
+<!--- the title includes "!" following the conventional commit type (e.g. "feat!: ..."-->
+None
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+- [x] I have performed a self-review of my own code
```

Comment:
I think the point of having this checkpoints is that you need always go through it make sure you did everything. So let's keep them unchecked.

Reply:
This is just copied from [the Platform repo](https://github.com/dashevo/platform/blob/v0.22-dev/scripts/release/pr_description.md), but OK, I will update it.